src/NuGet.Clients/NuGet.VisualStudio.Client/ContentManifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json.schemastore.org/vsix-manifestinjection",
+  "urlAssociations": [
+    {
+      "protocol": "nuget-client",
+      "displayName": "Visual Studio NuGet Deep Link URI Handler",
+      "progId": "VisualStudio.nuget-client.[InstanceId]",
+      "defaultProgramRegistrationPath": "Software\\Microsoft\\VisualStudio_[InstanceId]\\Capabilities"
+    }
+  ],
+  "progIds": [
+    {
+      "id": "VisualStudio.nuget-client.[InstanceId]",
+      "displayName": "Visual Studio URI Handler",
```

Comment:
In a previous discussion we talked about how this should probably reflect the NuGet Deep Link URI handler but that you should check with the experts in protocol handling. Did you have a chance to check on this?